src/equations/shallow_water_1d.jl

```diff
@@ -472,6 +529,25 @@ end
 end
 
 
+@inline function min_max_speed_chen_noelle(u_ll, u_rr, orientation::Integer, equations::ShallowWaterEquations1D)
```

Comment:
add a docstring to describe the motivation of the this min/max eigenvalue estimate.